crates/tuono/src/scaffold_project.rs

```diff
@@ -292,4 +329,25 @@ mod tests {
         );
         assert_eq!(expected, generated)
     }
+
+    #[test]
+    fn test_init_new_git_repo() {
```

Comment:
I'd remove this test. The mock is not good enough since the success/fail of the test depends on the running machine (If `git` is not installed it fails)